test/statewide_test_repository_test.rb

```diff
@@ -8,5 +8,72 @@ def test_starts_off_with_empty_statewide_tests
   end
 
   def test_can_load_data
+    str = StatewideTestRepository.new
+    str.load_data({
+      statewide_testing: {
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.